docs/exploration-and-organization/content-verification.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Content verification
+---
+
+# Content verification
+
+{% include plans-blockquote.html feature="Content verification" %}
+
+![Verified icon](./images/verified-icon.png)
+
+Admins can verify items to let others know that the content is accurate and up to date with the correct data source and query.
```

Comment:
Also dashboards don't have a single data source and query, so this might need to be plural